README.md

```diff
@@ -0,0 +1,71 @@
+# To-Do List
+This is minimalist To-Do app. It allows the user to stay organized by offering the following possibilies:
+- Possibility to create a list of tasks 
+- Possibility to mark them as completed
+- Possibility to remove a task
+- Possibility to re-order tasks
+- Possibility to remove all completed tasks
+
+## Screenshot
+
+![Screenshot](./To-do-list.png)
+
+## Live demo
+
+Click [here](https://mugberto.github.io/to-do-list/) to view
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps inside the root folder.
+
+### Setup
+
+Install dependencies with:
+
+```
+npm run build
```

Comment:
The command to install dependencies is `npm install`.

`npm run build` will run the `webpack` command instead.